vela-templates/definitions/internal/trait/container-ports.cue

```diff
@@ -0,0 +1,132 @@
+import (
+	"strconv"
+	"strings"
+)
+
+"container-ports": {
+	type: "trait"
+	annotations: {}
+	labels: {}
+	description: "Add hostPort and hostIP for the specified container of K8s pod for your workload which follows the pod spec in path 'spec.template'."
```

Comment:
This trait support not only the hostPort but also regular container port (by skipping the hostPort). We should explain when you should use `expose` and `gateway` traits and when to use `container-ports`. 
Another thing, in my point of view this trait is dedicated for `hostPort` so we can rename the trait to `host-ip`. Then remind user to avoid use this unless he knows what he's doing.

Reply:
done